### PR TITLE
[#2941] Move start in the control-center app

### PR DESCRIPTION
**Pull request by ljupcovangelski**
Making the start page as part of the `control-center`, instead of it running in a separate pod.
The root uri redirects to `/control-center/start.html` and from there one can navigate to either `control-center` or `/inbox`.

### CODE REVIEW

**Review by chrismatix — DISMISSED** [review]
Very good. One less container.

File: frontend/control-center/public/start.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Airy Start Page</title>
+  <meta charset="utf-8" />
+  <link rel="shortcut icon" href="/favicon.ico" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <meta name="theme-color" content="#000000" />
+  <link rel="manifest" href="/manifest.json" />
+  <link href="http://fonts.googleapis.com/css?family=Lato" rel="stylesheet" type="text/css">
```
Comment (chrismatix) [inline frontend/control-center/public/start.html:10]: @AitorAlgorta This is where the import needs to be fixed. The original `start.html` was moved here :)

File: frontend/start/index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
   <meta name="theme-color" content="#000000" />
   <link rel="manifest" href="/manifest.json" />
-  <link href="http://fonts.googleapis.com/css?family=Lato" rel="stylesheet" type="text/css">
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet" type="text/css">
```
Comment (chrismatix) [inline frontend/start/index.html:10]: @AitorAlgorta So this file was meant to be deleted

File: frontend/control-center/public/start.html (hunk 1 above)
Comment (AitorAlgorta) [reply frontend/control-center/public/start.html]: You right!

**Review by AitorAlgorta — APPROVED** [review]

**Pull request merged**